ReactiveDataDisplayManagerTests/REFACTORING/Table/Mock/Generators/MockTableCellGenerator.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MockTableCellGenerator.swift
+//  ReactiveDataDisplayManagerTests
+//
+//  Created by Anton Eysner on 08.02.2021.
+//  Copyright © 2021 Александр Кравченков. All rights reserved.
+//
+import UIKit
+
+@testable import ReactiveDataDisplayManager
+
+class MockTableCellGenerator: TableCellGenerator {
```

Comment:
```suggestion
class StubTableCellGenerator: TableCellGenerator {
```

Ничего не подменяем и не шпионим, поэтому это всего лишь Stub. Даже не Spy.